app/views/inventory_tallies/_form.html.erb

```diff
@@ -13,7 +13,7 @@
   <div class="form-inputs container">
       <%= f.association :inventory_type %>
       <%= f.association :storage_location, collection: Location.storage_unit %>
-      <%= f.text_area :additional_location_info %>
+      <%= f.input :additional_location_info %>
```

Comment:
Hey there, @smoline !

This definitely solves the test issue, but we lose the larger "text area" box. Would you change this input to something like the following?

```
<%= f.input : additional_location_info,
                              as: :text,
                              input_html: {value: f.object. additional_location_info",
                                           rows: 2} %>
```